src/components/QuranReader/TranslationView/TranslationViewCell.module.scss

```diff
@@ -1,7 +1,26 @@
 @use "src/styles/breakpoints";
 
 .highlightedContainer {
-  background: var(--color-background-alternative-faded);
+  --color-gradient-transition: 196, 196, 196;
+
+  [data-theme="dark"] & {
```

Comment:
We have a partial called `_theme.scss` that support dark and light themes. To use this, please refer to `Skeleton.module.scss`lines 19 and 22.

<img width="653" alt="Screen Shot 2021-12-20 at 18 57 24" src="https://user-images.githubusercontent.com/15169499/146763819-4b426715-5273-4f15-9d82-27aebd0bd133.png">